src/sage/combinat/posets/all.py

```diff
@@ -36,8 +36,8 @@
 from sage.misc.namespace_package import install_doc
 install_doc(__package__, __doc__)
```

Comment:
At some point in this file: `del install_doc`.

Reply:
Done throughout in 8de494c640b